Projects/Hangman Game.py

```diff
@@ -0,0 +1,72 @@
+
+randomWord = random.choice(words)
+print("Hangman Game!")
```

Comment:
This line can have the new line command instead of a new print statement.
print("Hangman Game!\n")